TRB_label_assist/clustering.py

```diff
@@ -28,9 +29,26 @@
     'mean_shift'
 ]
 
+def CleanEntryTypeData(loc_df,loc_entry):
+
+    """
+    Helps weed out entries from the list of entries who's which were removed from the df using
+    esdtq.filter_labeled_trips() and esdtq.expand_userinputs()
+
+    loc_df : dataframe amde from entry type data
+    loc_entry : the entry type equivalent of loc_df ,
+                which was passed alongside the dataframe while loading the data
+
+    """
+
+    ids_in_df=set(loc_df['_id'])
```

Comment:
why do you need to use `set`? The `_id` entries are database entries and are guaranteed to be unique.
(as an aside, even if they are not unique, I believe `entry['_id'] in ids_in_df` will still work - it doesn't require a unique list IIRC)

Reply:
used set just for faster lookup (at least on average) as compared to list. Even without it, this will work

Reply:
But, as I said, these are guaranteed to be unique. So wrapping them in a set is unnecessary overhead.

Reply:
This has not yet been resolved